ui/plugins/controller.js

```diff
@@ -339,7 +339,20 @@ treeherder.controller('PluginCtrl', [
                    ($scope.job.state === "pending" || $scope.job.state === "running");
         };
 
-        $scope.retriggerJob = function (jobs) {
+        $scope.spinButton = function (selector, spinclass) {
```

Comment:
I don't see the need to create a function for this-- just inline this logic in `$scope.retriggerJob` below.